changes/2953.feature.rst

```diff
@@ -0,0 +1 @@
+Added a layout debugging mode that sets containers to random background colors.
```

Comment:
The change note is the draft of the release note for the feature, rather than being a commit summary; so - it should be worded in terms of "exiting new feature you can now use!!1!!BBQ!":
```suggestion
Toga now has a layout debugging mode. If you set ``TOGA_DEBUG_LAYOUT=1`` in your app's runtime environment, widgets will be rendered with different background colors, making it easier to identify how space is being allocated by Toga's layout algorithm.
```